### ISSUE TITLE
프로젝트 repository 생성한 사람은 프로젝트에 어떻게 참여해야 하나요?

### ISSUE THREAD

**Issue by CookieHCl** [snu-sf-class/swpp202101#35]
자기 자신의 repository는 fork가 안 되던데 프로젝트 repository를 생성한 사람은 같은 repository에서 따로 branch를 만들어서 참여하는 방법밖에 없나요?

**Comment by aqjune** [comment]
네, 같은 repository에 branch를 만드신 후에 pull request를 생성하는 방식으로 참여 부탁드립니다.
